src/Image.php

```diff
@@ -117,6 +117,9 @@ public function getIDCMetadata(
                 'radius' => $region->rbRx,
 
                 'vertices' => [],
+
+                'regionDefinitionId' => $region->regionDefinitionId,
+                'regionName' => $region->regionName,
```

Comment:
I'm tempted to return these values only when `$essentialOnly` is false. Would this work for you @klaari? Thoughts @ilu?

Reply:
Sure, works fine with me. I can do the update if we all agree?

Reply:
Works for me @AurelienLourot

Reply:
@klaari yes you can do the update, thanks a lot